llama.cpp

```diff
@@ -2868,19 +2877,19 @@ struct llama_model_loader {
 
     llama_mmaps mappings;
 
-    // Holds information on a model weights
-    struct llama_tensor_weights {
+    // Holds information on a model weight
+    struct llama_tensor_weight {
```

Comment:
Thanks

Reply:
I guess it could be right both ways depending on how you look at it, as the tensors typically represent an an entire layer and not a single weight, but it looked weird to me.